### DISCUSSION TITLE
Example implementation of the indexed db persister

### DISCUSSION THREAD

**Discussion by kinderlydev** [TanStack/query#5646] [Q&A]
Hello, 
Although I have read all the docs relating to building a persistor, can any body point me to an example implentation of indexed db, so i can see how its implemented, i'm not fully getting it from reading the docs.

**Comment by kinderlydev** [comment]
No one have a working code sandbox example maybe? Would be really helpful.

**Comment by TkDodo** [comment]
it's even in the docs: https://tanstack.com/query/v4/docs/react/plugins/persistQueryClient#building-a-persister

**Comment by kinderlydev** [comment]
Ah thanks, i've already seen this, but didn't have much luck implementing it. oh well. thanks anyways.
